onnxruntime/python/tools/quantization/quant_utils.py

```diff
@@ -202,6 +212,35 @@ def quantize_nparray(qType, arr, scale, zero_point, low=None, high=None):
         )
         ref = ReferenceEvaluator(onnx_model)
         return _check_type(ref.run(None, {"X": arr, "scale": scale})[0])
+    elif qType in (
```

Comment:
Since these data types were only introduced in opset 21, the quantization script should check the opset version of the model https://github.com/microsoft/onnxruntime/blob/69cfcba38a60d65498f94cde30cb9c2030f7255b/onnxruntime/python/tools/quantization/base_quantizer.py#L167

Reply:
Thanks @jambayk. I added checks in the files quantization/qdq_quantizer.py and quantization/execution_providers/qnn/quant_config.py

Reply:
we should also check this in the base_quantizer.py